hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/logaggregation/TestLogAggregationService.java

```diff
@@ -252,7 +252,7 @@ private void verifyLocalFileDeletion(
         count = 0;
         while ((f.exists()) && (count < maxAttempts)) {
           count++;
-          Thread.sleep(100);
+          Thread.sleep(1000);
```

Comment:
Is there some scope to change this hard-coded sleep with ``GenericTestUtils.waitFor``?

Reply:
Thank you very much for your help reviewing the code,  I will use `GenericTestUtils.waitFor` for code refactoring.